src/Sauce/Sausage/SauceAPI.php

```diff
@@ -53,6 +53,15 @@ protected function makeRequest($url, $type="GET", $params=false)
         if (getenv('CURL_CA_BUNDLE')) {
             curl_setopt($ch, CURLOPT_CAINFO, getenv('CURL_CA_BUNDLE'));
         }
+        
+        // If user has requested it, be extremely verbose when making requests.
+        // This is primarily intended to help Sauce Support staff figure out what's
+        // busted.
+        if (getenv('SAUCE_DIAGNOSE_SSL')) {
+            curl_setopt($ch, CURLOPT_CERTINFO, true);
+            curl_setopt($ch, CURLOPT_VERBOSE, true);
+            curl_setopt($ch, CURLOPT_STDERR, getenv('SAUCE_DIAGNOSE_SSL'));
```

Comment:
what's the purpose of not passing `true` here? is there a text value that gets sent in for this?